tendrl/integrations/ceph/sds_sync/__init__.py

```diff
@@ -0,0 +1,108 @@
+import gevent
+import json
+import os
+
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message, ExceptionMessage
+from tendrl.commons.utils import cmd_utils
+from tendrl.commons import sds_sync
+from tendrl.node_agent.node_sync import disk_sync
+
+
+class CephIntegrtaionsSyncThread(sds_sync.StateSyncThread):
+    def _run(self):
+        Event(
+            Message(
+                priority="info",
+                publisher=NS.publisher_id,
+                payload={"message": "%s running" % self.__class__.__name__}
+            )
+        )
+        while not self._complete.is_set():
+            # sync of OSD journal details every 30 sec is fine
+            # as this data doesnt change very frequently in cluster
+            gevent.sleep(30)
+            if NS.tendrl_context.integration_id is not None:
+                try:
+                    # Sync the OSD journal details
+                    Event(
```

Comment:
Please use https://github.com/Tendrl/commons/blob/develop/tendrl/commons/utils/log_utils.py instead of Event/Message